src/hexen/p_enemy.c

```diff
@@ -4946,6 +4946,7 @@ void A_KoraxChase(mobj_t * actor)
             P_Teleport(actor, spot->x, spot->y, spot->angle, true);
         }
 
+        CheckACSPresent(249);
```

Comment:
I don't have a problem doing this if there's a define for each of the Korax scripts; the next line after each CheckACSPresent call hard codes the same number too.